aspnetcore/blazor/state-management/server.md

```diff
@@ -0,0 +1,235 @@
+
+When a user can't be reconnected to their original circuit, the user receives a new circuit with newly initialized state. This is equivalent to closing and reopening a desktop app.
+
+## When to persist user state
+
+State persistence isn't automatic. You must take steps when developing the app to implement stateful data persistence.
+
+Generally, maintain state across circuits where users are actively creating data, not simply reading data that already exists.
+
+Data persistence is typically only required for high-value state that users expended effort to create. Persisting state either saves time or aids in commercial activities:
+
+* Multi-step web forms: It's time-consuming for a user to re-enter data for several completed steps of a multi-step web form if their state is lost. A user loses state in this scenario if they navigate away from the form and return later.
+* Shopping carts: Any commercially important component of an app that represents potential revenue can be maintained. A user who loses their state, and thus their shopping cart, may purchase fewer products or services when they return to the website later.
+
+An app can only persist *app state*. UIs can't be persisted, such as component instances and their render trees. Components and render trees aren't generally serializable. To persist UI state, such as the expanded nodes of a tree view control, the app must use custom code to model the behavior of the UI state as serializable app state.
+
+:::moniker range=">= aspnetcore-10.0"
+
+## Circuit state persistence
+
+During server-side rendering, Blazor Web Apps can persist a user's session (circuit) state when the connection to the server is lost for an extended period of time or proactively paused, as long as a full-page refresh isn't triggered. This allows users to resume their session without losing unsaved work in the following scenarios:
+
+* Browser tab throttling
+* Mobile device users switching apps
+* Network interruptions
+* Proactive resource management (pausing inactive circuits)
+
+<!-- UPDATE 10.0 - Arriving for Preview 7 ...
+
+                   `[SupplyParameterFromPersistentComponentState]` will be renamed to `[PersistentState]`.
+                   `Blazor.pauseCircuit` will be renamed to `Blazor.pause`.
+                   `Blazor.resumeCircuit` will be renamed to `Blazor.resume`.
+
+                   Guidance changes for persistent component state with enhanced nav
+                   will be on the upcoming docs PR for Preview 7.
+
+                   API review for support persistent component state on enhanced navigation
+                   https://github.com/dotnet/aspnetcore/issues/62773
+                   -->
+
+*[Enhanced navigation](xref:blazor/fundamentals/routing#enhanced-navigation-and-form-handling) with circuit state persistence isn't currently supported but planned for a future release.*
+
+Persisting state requires fewer server resources than persisting circuits:
```

Comment:
I don't really get what's being compared here. It's not like we have support for persisting entire circuits. I think the intended point is that pausing circuits by persisting and evicting them saves server resources. Maybe something like this?:

```suggestion
Server resources can be freed up if the circuit state can be persisted and then resumed later:
```